examples/space/simple_rn.py

```diff
@@ -88,23 +88,23 @@ def asarray(self, *args):
             return self.data(*args)
 
 r5 = SimpleRn(5)
-odl.diagnostics.SpaceTest(r5).run_tests()
+#odl.diagnostics.SpaceTest(r5).run_tests()
 
 # Do some tests to compare
 n = 10**7
 iterations = 10
 
 # Perform some benchmarks with Rn adn CudaRn
-opt_spc = odl.Rn(n)
+opt_spc = odl.rn(n)
 simple_spc = SimpleRn(n)
 
 x, y, z = np.random.rand(n), np.random.rand(n), np.random.rand(n)
 ox, oy, oz = (opt_spc.element(x.copy()), opt_spc.element(y.copy()),
               opt_spc.element(z.copy()))
 sx, sy, sz = (simple_spc.element(x.copy()), simple_spc.element(y.copy()),
               simple_spc.element(z.copy()))
-if odl.CUDA_AVAILABLE:
-    cu_spc = odl.CudaRn(n)
+if 'cuda' in odl.FN_IMPLS:
```

Comment:
This is obviously more general, but perhaps not the prefered method of doing this. I'm not sure how to do it otherwise.